src/WCSimConstructCylinder.cc

```diff
@@ -2707,6 +2707,11 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructCylinderNoReplica()
 	<< "GEOMCHECK3 WCPMTODExposeHeight \t" << WCPMTODExposeHeight << G4endl
 	;
 #endif
+  // Tell DetectorConstruction about boundary walls
+  AddBoundaryWallCylinderDimensions(kBoundaryWallODInnerTyvek,
+									WCODRadius,
+									(WCIDHeight + 2*WCODDeadSpace)); //from CapTyvekPosition creation, without the /2 as we store the full length in the boundary wall variables
```

Comment:
should be `WCIDHeight + 2*WCODDeadSpace+WCODTyvekSheetThickness`, to be consistent with `WCODRadius` which is the surface touching OD water